resources/views/community/components/developer/sets-page.blade.php

```diff
@@ -0,0 +1,153 @@
+                    @else
+                        <td class='text-right'>{{ $game['NumAuthoredLeaderboards'] }} of {{ $game['leaderboards_count'] }}</td>
+                    @endif
+
+                    <td class='text-right'>{{ $game['players_total'] }}</td>
+
+                    @if ($game['NumTickets'] == 0)
+                        <td></td>
+                    @elseif ($game['NumAuthoredTickets'] == $game['NumTickets'])
+                        <td class='text-right'><a href="/ticketmanager.php?g={{ $game['ID'] }}">{{ $game['NumAuthoredTickets'] }}</a></td>
+                    @else
+                        <td class='text-right'><a href="/ticketmanager.php?g={{ $game['ID'] }}">{{ $game['NumAuthoredTickets'] }} of {{ $game['NumTickets'] }}</a></td>
+                    @endif
+
+                    @if ($userProgress !== null)
+                        <td>
+                        <?php
+                            $hardcoreProgressBarWidth = $softcoreProgressBarWidth = 0;
+                            $gameProgress = $userProgress[$game['ID']] ?? null;
+                            if ($gameProgress != null) {
+                                $hardcoreProgressBarWidth = sprintf("%01.2f", $gameProgress['achievements_unlocked_hardcore'] * 100 / $game['achievements_published']);
+                                $softcoreProgressBarWidth = sprintf("%01.2f", ($gameProgress['achievements_unlocked'] - $gameProgress['achievements_unlocked_hardcore']) * 100 / $game['achievements_published']);
+                            }
+                        ?>
+                        <div role="progressbar" aria-valuemin="0" aria-valuemax="100"
+                                class="w-full h-1 bg-embed rounded flex">
+                            <div style="width: {{ $hardcoreProgressBarWidth }}%"
+                                    class="bg-[#cc9900] h-full {{ $hardcoreProgressBarWidth > 0 ? 'rounded-l' : '' }}"></div>
+                            <div style="width: {{ $softcoreProgressBarWidth }}%"
+                                    class="bg-[rgb(11,113,193)] h-full {{ $hardcoreProgressBarWidth === 0 ? 'rounded-l' : '' }}"></div>
+                        </div>
+                        </td>
+                    @endif
+                </tr>
+            @endforeach
+            @if ($count > 1)
+                <tr>
+                    <td><b>Total:</b> {{ $count }} games</td>
+                    <td class='text-right'><b>{!! localized_number($achievementCount) !!}</b></td>
+                    <td class='text-right'><b>{!! localized_number($pointCount) !!}</b></td>
+                    <td></td>
+                    <td class='text-right'><b>{!! localized_number($leaderboardCount) !!}</b></td>
+                    <td></td>
+                    <td class='text-right'><b>{!! localized_number($ticketCount) !!}</b></td>
+                    @if ($userProgress !== null)
+                        <td></td>
```

Comment:
Average progress here would also be very interesting to see. I think this column as you've built it is really cool, this is another thing that could make it shine.